HappyAnding/HappyAnding/Views/ShortcutDetailViews/ReadShortcutView/ReadShortcutView.swift

```diff
@@ -232,10 +232,11 @@ extension ReadShortcutView {
                         .foregroundColor(.gray4)
                 }
                 TextField(useWithoutSignIn ? TextLiteral.readShortcutViewCommentDescriptionBeforeLogin : TextLiteral.readShortcutViewCommentDescription, text: $commentText, axis: .vertical)
```

Comment:
저는 과도한 줄바꿈이 오류로 인식될 것 같다는 의견에 동의합니다 ..! return키가 없는 키보드 타입을 적용하는 건 어떤가요? 태그 기능이 구현되어 있지 않아서 어색할까요 ???
```suggestion
                TextField(useWithoutSignIn ? TextLiteral.readShortcutViewCommentDescriptionBeforeLogin : TextLiteral.readShortcutViewCommentDescription, text: $commentText, axis: .vertical)
                .keyboardType(.twitter)
```

Reply:
저도 댓글에서는 크게 줄바꿈의 필요성을 못느껴서 키보드 타입을 변경하는 것도 좋은 의견이라고 생각합니다!